packages/e2e/src/page-objects/messages.page.ts

```diff
@@ -145,13 +145,14 @@ class MessagesPage extends Page {
   async getLastMessages() {
     const allLastMessages = await $$('//*[@data-testid="thread-last-message"]')
 
-
     const messagesContents: string[] = await browser.executeAsync((done) => {
+
```

Comment:
Are those a mistake or did you want just to add those line breaks?

Reply:
mistake indeed. Thx! I resolved conflicts using github web editor and did not check formatting afterwards..

Reply:
https://github.com/mudita/mudita-center/pull/1165/commits/1fa0d1921b263d1786d46b0618516b5fee3a7810